src/main.js

```diff
@@ -89,8 +109,13 @@ function getDOI (el) {
   return doi && doi.toLowerCase().trim()
 }
 
-export function insertBadge (el, tally, notices) {
+export function insertBadge (el, tally, notices, sectionTally) {
   const config = getConfig(el)
+
+  const showTally = config.showTally || false
```

Comment:
We will need to make this default to true to make it backward comparable.

Reply:
Oops, it should be true, I reverted it to debug nice catch